src/client/nosto.d.ts

```diff
@@ -813,7 +813,7 @@ interface Order {
     items: CartItem[];
 }
 
-interface PluginMetadata {
+export interface PluginMetadata {
```

Comment:
are these changes auto-generated?

Reply:
the client script changes are here https://github.com/Nosto/playcart/pull/33521